src/main/java/edu/ucla/library/prl/harvester/Institution.java

```diff
@@ -148,9 +148,9 @@ public Institution(final String aName, final String aDescription, final String a
             final Optional<InternetAddress> anEmail, final Optional<PhoneNumber> aPhone,
             final Optional<URL> aWebContact, final URL aWebsite) {
         myID = Optional.empty();
-        myName = Objects.requireNonNull(aName);
-        myDescription = Objects.requireNonNull(aDescription);
-        myLocation = Objects.requireNonNull(aLocation);
+        myName = Objects.requireNonNull(aName.trim());
+        myDescription = Objects.requireNonNull(aDescription.trim());
+        myLocation = Objects.requireNonNull(aLocation.trim());
```

Comment:
Might as well use StringUtils#trimToNull here, so that strings containing only whitespace are rejected?